crates/topos/tests/config.rs

```diff
@@ -221,5 +218,7 @@ async fn test_command_init_precedence_cli_env() -> Result<(), Box<dyn std::error
     assert!(config_contents.contains("role = \"sequencer\""));
     assert!(config_contents.contains("subnet = \"topos-cli\""));
 
+    remove_dir_all("/tmp/topos/test_command_init_precedence_cli").unwrap();
```

Comment:
shouldn't it be `tmp_home_dir`?

Reply:
No, we use the `tmp_home_dir` just for the polygon edge install in this case. I removed this hand-set folder, because if you re-run the tests locally, it fails because the folder is not automatically removed.